Sentry.AWSLambda.init doesn't work with Netlify dev

I'm using `@sentry/serverless@5.27.1` ([docs](https://docs.sentry.io/platforms/node/guides/aws-lambda/)) with Netlify Functions (@netlify). It works great when deployed to Netlify, however when running locally with `> netlify dev` it fails with the following error:

> {"errorMessage":"Cannot find module 'aws-sdk/global'\nRequire stack:\n- C:\\Users...

Simple workaround incase others run into the issue:

```js
const Sentry = require("@sentry/serverless");
if (process.env.NETLIFY_DEV === "true") {
  Sentry.init(settings);
} else {
  Sentry.AWSLambda.init(settings);
}
```

On a side note, the `Sentry.AWSLambda.wrapHandler` works fine with Netlify dev.

--- a/packages/serverless/src/awslambda.ts
+++ b/packages/serverless/src/awslambda.ts
@@ -44,7 +44,7 @@ export interface WrapperOptions {
   timeoutWarningLimit: number;
 }
 
-export const defaultIntegrations: Integration[] = [...Sentry.defaultIntegrations, new AWSServices()];
+export const defaultIntegrations: Integration[] = [...Sentry.defaultIntegrations, new AWSServices({ optional: true })];
 
 /**
  * @see {@link Sentry.init}

--- a/packages/serverless/src/awsservices.ts
+++ b/packages/serverless/src/awsservices.ts
@@ -27,46 +27,58 @@ export class AWSServices implements Integration {
    */
   public name: string = AWSServices.id;
 
+  private readonly _optional: boolean;
+
+  public constructor(options: { optional?: boolean } = {}) {
+    this._optional = options.optional || false;
+  }
+
   /**
    * @inheritDoc
    */
   public setupOnce(): void {
-    const awsModule = require('aws-sdk/global') as typeof AWS;
-    fill(
-      awsModule.Service.prototype,
-      'makeRequest',
-      <TService extends AWSService, TResult>(
-        orig: MakeRequestFunction<GenericParams, TResult>,
-      ): MakeRequestFunction<GenericParams, TResult> =>
-        function(this: TService, operation: string, params?: GenericParams, callback?: MakeRequestCallback<TResult>) {
-          let transaction: Transaction | undefined;
-          let span: Span | undefined;
-          const scope = getCurrentHub().getScope();
-          if (scope) {
-            transaction = scope.getTransaction();
-          }
-          const req = orig.call(this, operation, params);
-          req.on('afterBuild', () => {
-            if (transaction) {
-              span = transaction.startChild({
-                description: describe(this, operation, params),
-                op: 'aws.request',
-              });
-            }
-          });
-          req.on('complete', () => {
-            if (span) {
-              span.finish();
-            }
-          });
-
-          if (callback) {
-            req.send(callback);
-          }
-          return req;
-        },
-    );
+    try {
+      const awsModule = require('aws-sdk/global') as typeof AWS;
+      fill(awsModule.Service.prototype, 'makeRequest', wrapMakeRequest);
+    } catch (e) {
+      if (!this._optional) {
+        throw e;
+      }
+    }
   }
+}
+
+/** */
+function wrapMakeRequest<TService extends AWSService, TResult>(
+  orig: MakeRequestFunction<GenericParams, TResult>,
+): MakeRequestFunction<GenericParams, TResult> {
+  return function(this: TService, operation: string, params?: GenericParams, callback?: MakeRequestCallback<TResult>) {
+    let transaction: Transaction | undefined;
+    let span: Span | undefined;
+    const scope = getCurrentHub().getScope();
+    if (scope) {
+      transaction = scope.getTransaction();
+    }
+    const req = orig.call(this, operation, params);
+    req.on('afterBuild', () => {
+      if (transaction) {
+        span = transaction.startChild({
+          description: describe(this, operation, params),
+          op: 'aws.request',
+        });
+      }
+    });
+    req.on('complete', () => {
+      if (span) {
+        span.finish();
+      }
+    });
+
+    if (callback) {
+      req.send(callback);
+    }
+    return req;
+  };
 }
 
 /** Describes an operation on generic AWS service */

